src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/DependencyNode.cs

```diff
@@ -266,7 +268,7 @@ public void SetProperties(string caption = null)
 
         public override int GetHashCode()
         {
-            return unchecked(Id.GetHashCode() + Resolved.GetHashCode());
+            return unchecked(Id.GetHashCode());
```

Comment:
Why not?

---
In reply to: [99721396](https://github.com/dotnet/roslyn-project-system/pull/1469#discussion_r99721396) [](ancestors = 99721396)